CHANGELOG-14.0.md

```diff
@@ -35,6 +35,38 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 <!-- Add generated changelog below this line -->
 <!-- Release notes generated using configuration in .github/release.yml at 14.0 -->
 
+## [v14.0.1](https://github.com/shopsys/shopsys/compare/v14.0.0...v14.0.1) (2024-09-10)
+
+### :sparkles: Enhancements and features
+
+-   [framework] improved formatting of Entity logs by @TomasLudvik in https://github.com/shopsys/shopsys/pull/3410
+
+### :bug: Bug Fixes
+
+-   [shopsys] moved two attributes of ProductFormType from project-base to framework by @stanoMilan in https://github.com/shopsys/shopsys/pull/3320
+-   [shopsys] Moved CSRFExtension from project-base to framework by @stanoMilan in https://github.com/shopsys/shopsys/pull/3318
```

Comment:
nitpick
some bullet points start with uppercase letters, while others start with lowercase. It would be visually more pleasing to unify it 😇